about.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta name="Description" content="INF 7420 Project">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Kendralyn Burr - Portfolio</title>
+</head>
+<body>
+  <header>
+    <nav>
+      <ul>
+        <li><a href="home.html">Home</a></li>
+        <li><a href="about.html">About</a></li>
+        <li><a href="works.html">Works</a></li>
+      </ul>
+    </nav>
+  </header>
+  <main>
+    <div><img src="images/desktop.jpg"></div>
+    <div>
+      <h1 id="section-title">Section-Title</h1>
+      <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Et magnis dis parturient montes nascetur ridiculus mus mauris vitae. Vitae purus faucibus ornare suspendisse. Nullam non nisi est sit amet facilisis. In ante metus dictum at. Pulvinar pellentesque habitant morbi tristique. Sed velit dignissim sodales ut eu. Diam ut venenatis tellus in metus vulputate eu. Cursus risus at ultrices mi tempus imperdiet nulla. Quis eleifend quam adipiscing vitae proin sagittis nisl rhoncus mattis. Enim lobortis scelerisque fermentum dui faucibus in ornare quam viverra. Odio facilisis mauris sit amet massa. Elit duis tristique sollicitudin nibh sit. Egestas congue quisque egestas diam in arcu cursus euismod quis. Porttitor leo a diam sollicitudin tempor.</p>
+
+      <h1 id="section-title-two">Section-Title-Two</h1>
```

Comment:
There should only be 1 `<h1>` per page and they generally describe the content for the page. I recommend adding an `<h1>About</h1>` and changing the existing `<h1>` tags to `<h2>`. The pattern you have in your `index.html` is a good one to follow for headings.